pages/common/locale.md

```diff
@@ -0,0 +1,12 @@
+# locale
```

Comment:
@einverne we raise a PR against `main` instead of `master`.
I think there may have been an innocent typo. I changed the base to `main`

Reply:
Oh, sorry, I use `hub pull-request` , I am not sure how `hub` command handles this.

Reply:
I use the github [cli/cli](https://github.com/cli/cli). 
When i create a PR, I do something like this: `gh pr create -B main`

On similar lines, `hub` has the same option: `hub pull-request -b main`

The only difference is the case of the switch `-b/-B` .
I think, we should add it in the `hub` & `gh` page. Let me send a PR for the same, if we do not have it documented already.